snapshots/devmapper/README.md

```diff
@@ -0,0 +1,42 @@
+## Devmapper snapshotter
+
+Devmapper is a `containerd` snapshotter plugin that stores snapshots in ext4-formatted filesystem images
+in a devicemapper thin pool.
+
+## Setup
+
+To make it work you need to prepare `thin-pool` in advance and update containerd's configuration file.
```

Comment:
Try and find it is not friendly to user if not familiar with devicemapper , take me some time to run with devicemapper snapshot, so give some example to describe `prepare thin-pool` will better